app/controllers/reservations_controller.rb

```diff
@@ -99,6 +99,9 @@ def create
       @reservation = creator.reservation
       flash[:notice] = I18n.t "controllers.reservations.create.success"
 
+      should_show_admin_hold_warning = creator.reservation.conflicting_admin_reservation.present?
+      flash[:error] = I18n.t('controllers.reservations.create.admin_hold_warning') if should_show_admin_hold_warning
```

Comment:
Double quotes. I don't have a strong feeling on this, but I probably wouldn't use the intermediate variable.

I think it might be worth adding a `Reservation#conflicting_admin_reservation?` method to make this and the other places a little shorter.